frontend/src/app/modules/posts/sevices/comment.service.ts

```diff
@@ -0,0 +1,30 @@
+import {Injectable, signal} from '@angular/core';
+import {finalize, Observable, tap} from "rxjs";
+import {HttpClient, HttpHeaders, HttpParams} from "@angular/common/http";
+import {AlertService} from "../../../shared/services/alert.service";
+import {CommentGetModel} from "../../../shared/types/posts/comment-get.model";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CommentService {
+
+  constructor(private http: HttpClient, private alertService: AlertService) { }
+
+  isLoadingGet = signal(false);
+  getComments(postId: string, userId: string): Observable<CommentGetModel[]> {
+    this.isLoadingGet.set(true);
+    const headers = new HttpHeaders().set('UserId', userId);
+    let params = new HttpParams().set('postId', postId);
+    return this.http.get<any>('http://localhost:8080/api/posts/comments', {headers, params}).pipe( //tez nie jestem pewny czy tak powinien wygladac endpoint
```

Comment:
trzeba dodać ogólnie do paginacji parametry do tej funkcji.  Ostatecznie chyba chciałbym żebyśmy właśnie po 10 pobierali, z czego te 2 pierwsze to byłyby chyba od razu pobrane przy getowaniu postu. (trzeba będzie z backendem zagadać)

Reply:
a jak wtedy z chowaniem komentarzy ?

Reply:
kurde nie jestem pewien, można by pominąć? albo zrobić żeby to cie przenosiło na stronę tylko tego posta? i tam pokazywało komentarze? albo po prostu w prawym rogu dajesz ikonke do collapsowania i chuj

Reply:
te drugie rozwiazanie to mega pracochłonne by było, te ostatnie wydaje sie git

Reply:
no to ok, najwyżej potem będziemy zmieniać